### DISCUSSION TITLE
Matter fan demo is not working

### DISCUSSION THREAD

**Discussion by yogeshiggalore** [SiliconLabs/arduino#38] [Q&A]
Hi all, I am working on the fan demo code with the Sparkfun matter-supported board. I did all the processes required to commission the hardware into the Alexa Echo. the device is visible in the Alexa app but does not take any commands. required your help here

**Comment by silabs-bozont** [comment]
Hi,
It looks like Matter Fan is currently not supported by Amazon according to their docs: https://developer.amazon.com/en-US/docs/alexa/smarthome/matter-support.html
Hopefully they'll add it sometime in the future.

**Comment by lboue** [comment]
Yes. Matter-enabled **Echo devices support the Matter 1.0 protocol**, defined by the [Connectivity Standards Alliance (CSA)](https://csa-iot.org/). For details, see the [Matter 1.0 specifications](https://csa-iot.org/developer-resource/specifications-download-request/).

Fan device type was introduced with Matter 1.2.

**Comment by Anzalone394** [comment]
hello

I reached out to you in order to have a better chat after seeing your article.
Are you still in need of assistance? If so, please let me know so that we may have a more in-depth conversation.
message me on discord: bushraa03

**Comment by Anzalone394** [comment]
hello

I reached out to you in order to have a better chat after seeing your article.
Are you still in need of assistance? If so, please let me know so that we may have a more in-depth conversation.
message me on discord: bushraa03
